base_app/base.py

```diff
@@ -10,6 +10,10 @@
 
 app = Flask(__name__)
 
+@app.route('/', methods=['GET'])
+def home():
+    return "<h1>Welcome to the Hangry-8Ball API!</h1><p>Enter API documentation and endpoints here.</p>"
```

Comment:
This is a great placeholder. We can start updating this with new features after this initial set is complete.